mumblepahelper.cpp

```diff
@@ -162,3 +162,44 @@ void MumblePAHelper::on_action_Quit_triggered(bool) {
 void MumblePAHelper::on_action_Rescan_triggered(bool) {
     plugins->rescanPlugins();
 }
+
+void MumblePAHelper::on_action_SystemPluginsLocation_triggered(bool) {
+	QString SystemPluginsLocation = QFileDialog::getExistingDirectory(this, tr("Select system plugins directory"), plugins->qsSystemPlugins, QFileDialog::ShowDirsOnly | QFileDialog::DontResolveSymlinks);
```

Comment:
The text in the dialog ("Select system plugins directory") should match the text on the action/menu item.
So this should probably read "Set system plugin directory"...